README.md

```diff
@@ -20,46 +26,114 @@ This library requires:
+[deadline-cloud-monitor-setup]: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/submitter.html#install-deadline-cloud-monitor
+[aws-cli-credentials]: https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-authentication.html
+
+### Maya Adaptor
+
+Jobs created by this submitter require this adaptor be installed on your worker hosts, and that both the installed adaptor
+and the Maya executable be available on the PATH of the user that will be running your jobs.
+
+The adaptor application is a command-line Python-based application that enhances the functionality of Maya for running
+within a render farm like Deadline Cloud. Its primary purpose for existing is to add a "sticky rendering" functionality
+where a single process instance of Maya is able to load the scene file and then dynamically be instructed to perform
+desired renders without needing to close and re-launch Maya between them, though it also supplies additional benefit
+such as support for path mapping, and reporting the progress of your render to Deadline Cloud. The alternative 
+to "sticky rendering" is that Maya would need to be run separately for each render that is done, and close afterwards.
+Some scenes can take 10's of minutes just to load for rendering, so being able to keep the application open and loaded between
+renders can be a significant time-saving optimization; particularly when the render itself is quick.
+
+If you are using the [default Queue Environment](default-queue-environment), or an equivalent, to run your jobs, then the adaptor will be
+automatically made available to your job. Otherwise, you will need to install the adaptor.
 
 The adaptor can be installed by the standard python packaging mechanisms:
 ```sh
 $ pip install deadline-cloud-for-maya
 ```
 
-After installation it can then be used as a command line tool:
+After installation it can then you can test that it has been installed properly by running the following as the same
+user that runs your jobs:
 ```sh
 $ maya-openjd --help
 ```
 
 For more information on the commands the OpenJD adaptor runtime provides, see [here][openjd-adaptor-runtime-lifecycle].
 
-## Versioning
+### Maya Software Availability in AWS Deadline Cloud Service Managed Fleets
 
-This package's version follows [Semantic Versioning 2.0](https://semver.org/), but is still considered to be in its 
-initial development, thus backwards incompatible versions are denoted by minor version bumps. To help illustrate how
-versions will increment during this initial development stage, they are described below:
+You will need to ensure that the version of Maya that you want to run is available on the worker host when you are using
+AWS Deadline Cloud's [Service Managed Fleets](service-managed-fleets) to run jobs;
+hosts do not have any rendering applications pre-installed. The standard way of accomplishing this is described
```

Comment:
This is a bit misleading; software isn't preinstalled but their are available conda packages that the submitter's jobs will automatically pull in. (Right? Maybe we don't have a Maya conda package.)

Reply:
Right, software isn't preinstalled. That's what I meant by "hosts do not have any rendering applications pre-installed". The links in the subsequent sentences point the customer to the documentation on using conda to install apps, and the ones that we make available in the service's internal conda channel. 

I will add "... hosts do not have any rendering applications pre-installed" to the end of the last sentence in the paragraph, though, to help make things a bit clearer. Does that help?